features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -565,6 +565,7 @@ function TryOutController(props) {
                                             defaultMessage='Enter access Token'
                                         />
                                     )}
+                                    id='accessTokenInput'
```

Comment:
why we need an ID here ?

Reply:
When we add the ID it generates the ` aria-describedby ` for accessibility